src/primitive_crypto.cc

```diff
@@ -156,27 +215,17 @@ PRIMITIVE(aes_cbc_init) {
 }
 
 PRIMITIVE(aes_cbc_crypt) {
-  ARGS(AesCbcContext, context, Blob, input, int, from, int, to, bool, encrypt);
-  if (from < 0 || to > input.length() || from > to || ((to - from) & 0xf) != 0) INVALID_ARGUMENT;
-
-  Error* error = null;
-  ByteArray* result = process->allocate_byte_array(to - from, &error);
-  if (result == null) return error;
-
-  ByteArray::Bytes output_bytes(result);
-  mbedtls_aes_crypt_cbc(
-      &context->context_,
-      encrypt ? MBEDTLS_AES_ENCRYPT : MBEDTLS_AES_DECRYPT,
-      to - from,
-      context->iv_,
-      input.address() + from,
-      output_bytes.address());
+  ARGS(AesContext, context, Blob, input, bool, encrypt);
+  return aes_crypt(process, context, &input, encrypt, AesTypes::Cbc);
+}
 
-  return result;
+PRIMITIVE(aes_ecb_crypt) {
+  ARGS(AesContext, context, Blob, input, bool, encrypt);
```

Comment:
```suggestion
  ARGS(AesCbcContext, context, Blob, input, bool, encrypt);
```

Reply:
Changed the context type for the aes_cbc_crypt. This led to needing an additional close primitive, which has been added in the VM and is now used in the aes lib.